src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java

```diff
@@ -277,15 +281,10 @@ else if (exercise instanceof ModelingExercise) {
      * @return returns the set of StudentExams of which the empty submissions were assessed
      */
     public Set<StudentExam> assessEmptySubmissionsOfStudentExams(final Exam exam, final User assessor, final Set<StudentExam> excludeStudentExams) {
```

Comment:
I noticed that this method is almost identical to `assessUnsubmittedStudentExams`. Is there a simple way to extract common code to a helper function?

Reply:
I extracted some code to helper functions :+1: